save-backend/src/main/kotlin/com/saveourtool/save/backend/service/UserDetailsService.kt

```diff
@@ -35,7 +35,7 @@ class UserDetailsService(
     private val lnkUserOrganizationRepository: LnkUserOrganizationRepository,
     private val lnkUserProjectRepository: LnkUserProjectRepository,
     private val avatarStorage: AvatarStorage,
-) {
```

Comment:
you suggested in previous PR to move logic related to COSV to separate module to simplify extraction of it in future.
COSV needs some API from backend, I decided to do it via interfaces in cosv module and implementations in backend.

Reply:
In future when COSV will be a separate application, it will changed to `BackendService` which makes rest requests to backend service (like preprocessor, orchestrator and etc)